src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -220,6 +220,11 @@ namespace {
                     this->snapshots.back().wells.update( well2 );
                 }
                 this->snapshots.back().wellgroup_events().addEvent( name, ScheduleEvents::COMPLETION_CHANGE);
+                const auto& md = connections->getMD();
+                if (!std::is_sorted(std::begin(md), std::end(md))) {
+                    auto msg = fmt::format("Well {} has no increasing measured depth column", name);
```

Comment:
Wouldn't be clearer to say something like: Well {} measured depth column in the WELLTRAJ keyword is not strictly increasing.

Reply:
That is better. I changed the error message. It gives now

```
Error: Problem with keyword WELTRAJ
In SPE1CASE1_TRAJ_WELSPEC_DEF.DATA line 410
Well INJ measured depth column is not strictly increasing

Error: Unrecoverable errors while loading input: Problem with keyword WELTRAJ
In SPE1CASE1_TRAJ_WELSPEC_DEF.DATA line 410
Well INJ measured depth column is not strictly increasing
```
The error message is repeated for some reason but I believe that this issue is addressed in https://github.com/OPM/opm-common/pull/3524